requirements.txt

```diff
@@ -1,3 +1,4 @@
 git+git://github.com/canonical/operator
 pyaml
-urllib3
+requests
+deepdiff
```

Comment:
Could you compare between `json.dumps({...}, sort_keys=True)` instead?
I think that avoiding a 3rd party dep is worth it considering the current use-case.

Reply:
I am kinda wary of nested datastructures. If we compares just the prometheus server, that could work though